site/en/blog/renderingng/index.md

```diff
@@ -0,0 +1,466 @@
+We put a ton of effort into RenderingNG to make sure that every possible scroll is threaded,
+through caching that goes well beyond just a display list to more complex situations.
+Examples include code to represent fixed- and sticky-positioned elements,
+passive event listeners, and high-quality LCD text rendering.
+
+{% Img src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/9qci9TU2xIEGshIPf8zZ.jpg",
+alt="Sketch shows that with RenderingNG performance stays solid.",
+width="800", height="559" %}
+
+### GPU acceleration
+
+A GPU makes generating pixels and drawing to the screen dramatically faster—
+in many cases, every pixel can be drawn in parallel with every other pixel,
+resulting in an enormous speed increase.
+A key component of RenderingNG is GPU raster/draw everywhere.
+This uses the GPU on all platforms, and all devices, to hyper-accelerate the rendering and animating of web content.
+This is especially a benefit on low-end devices,
+which often have a much more capable GPU than other parts of the device.
+
+GPU hardware acceleration is especially important for very high-resolution screens or low-end hardware.
+
+{% Img src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/2kly2kdoelhTeDqUHEPn.jpg",
+alt="Sketch shows that with RenderingNG performance does not degrade so much.",
+width="800", height="559" %}
+
+## Extensibility: the right tools for the job
+
+Once we have reliability and scalable performance,
+we're now ready to build on top a host of tools to help developers extend the built-in parts of HTML,
+CSS and Canvas,
+and in ways that do not sacrifice any of that hard-won performance and reliability.
+
+This includes built-in plus JavaScript-exposed APIs for advanced use cases of responsive design,
+progressive rendering, smoothness and responsiveness, and threaded rendering.
+
+The following open web APIs, championed by Chromium, were made possible by RenderingNG,
+and were previously considered infeasible.
+
+All of them were developed with open specifications and collaboration with open web partners—
+engineers at other browsers, experts, and web developers.
+In subsequent blog posts, we will dive into each of these and explain how RenderingNG makes them possible.
+
+- [content-visibility](https://web.dev/content-visibility/):
+allows sites to easily avoid rendering work for offscreen content,
+and cache rendering for not-currently-shown SPA views.
+- [OffscreenCanvas](/web/updates/2018/08/offscreen-canvas): allows canvas rendering
+(both the 2D canvas API and WebGL) to run on its own thread for reliably excellent performance.
+This project is also another major milestone for the web.
+It's the very first web API that allows JavaScript (or WebAssembly!)
+to render a single web page document from multiple threads.
+- [Container queries](https://web.dev/new-responsive/): allow a single component to responsively lay itself out.
+This unblocks a whole universe of plug-and-play components, and is currently an experimental implementation.
```

Comment:
```suggestion unblocking a whole universe of plug-and-play components. (This feature is currently an experimental implementation.)
```

Reply:
The suggesting change rendering looks, weird; what I want it to say is:

"unblocking a whole universe of plug-and-play components. (This feature is currently an experimental implementation)"